cts/cts-cli.in

```diff
@@ -2289,6 +2289,38 @@ function test_tools() {
     desc="Switch to nonexistent shadow instance (force) (XML)"
     cmd="crm_shadow --switch $shadow --batch --force --output-as=xml"
     test_assert_validate $CRM_EX_NOSUCH 0
+    
+    export CIB_file_invalid_1="$test_home/cli/crm_verify_invalid_bz.xml"
+    export CIB_file_invalid_2="$test_home/cli/crm_verify_invalid_no_stonith.xml"
+    export CIB_file_invalid_3="$test_home/cli/crm_verify_invalid_xml_error.xml"
```

Comment:
You don't need `export` for these since they are only used within this file. `export` is needed when the commands that are called need to use the variable themselves.